rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -771,7 +769,46 @@ type StrikeSpread: <"A class for defining a strike spread feature.">
     upperStrike OptionStrike (1..1) <"Upper strike in a strike spread.">
     upperStrikeNumberOfOptions number (1..1) <"Number of options at the upper strike price in a strike spread.">
 
+type Exercise: <"A class defining the exercise period for an option together with any rules governing the notional amount of the underlying which can be exercised on any given exercise date and any associated exercise fees.">
```

Comment:
Rename type as `ExerciseTerms` (same as how the attribute is named), because "Exercise" would typically refer to the event itself more than the terms under which it can be executed.

Reply:
Okay, I agree - done